libs/datamodel/connectors/dml/src/default_value.rs

```diff
@@ -127,6 +127,15 @@ impl DefaultValue {
     pub fn db_name(&self) -> Option<&str> {
         self.db_name.as_deref()
     }
+
+    // Returns the dbgenerated function for a default value
+    // intended for primary key values!
+    pub fn dbgenerated_func(&self) -> Option<String> {
```

Comment:
Rust has a few naming conventions, like `into` implying consuming the conversion value. `to` is used for options and reference conversions, so I'd suggest using `to_dbgenerated_func`. FYI https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv (we don't follow that structly of course). `as` could also work here, even though it doesn't perfectly align with the naming outlined in the document.

Reply:
That is actually a great link and suggestion, thanks! :+1: